lms/templates/instructor/instructor_dashboard_2/make_invoice_transaction_modal.html

```diff
@@ -0,0 +1,89 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+<%page args="section_data"/>
+<section id="make_invoice_transaction_modal" class="modal" role="dialog" tabindex="-1" aria-label="${_('Add Coupon')}">
+    <div class="inner-wrapper">
+        <button class="close-modal">
+            <i class="icon fa fa-remove"></i>
+      <span class="sr">
+        ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
+        ${_('Close')}
+      </span>
+        </button>
+
+        <div id="coupon-content">
+            <header>
+                <h2>${_("Add Coupon")}</h2>
+            </header>
+
+            <div class="instructions">
+                <p>
+                    ${_("Please enter Coupon detail below")}</p>
+            </div>
+
+            <form action="${ section_data['make_invoice_transaction_url'] }" id="frm_make_invoice_transaction"
+                  method="post">
+
+                <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
+                <div id="make_invoice_form_error" class="modal-form-error"></div>
+
+                <div id="coupon_form_error" class="make_invoice-modal-form-error"></div>
+                <fieldset class="group group-form group-form-requiredinformation">
+                    <legend class="is-hidden">${_("Required Information")}</legend>
+
+                    <ol class="list-input">
+                        <li class="field required text" id="make_invoice-modal-field-invoice-id">
+                            <label for="invoice_id" class="required">${_("Invoice Id")}</label>
+                            <input class="" id="invoice_id" type="text" name="invoice_id" maxlength="16" value=""
+                                   placeholder="example: 1234"
+                                   aria-required="true"/>
+                        </li>
+                        <li class="field required text" id="make_invoice-modal-field-amount">
+                            <label for="amount" class="required text">${_("Amount")}</label>
+                            <input class="field required" id="amount" type="text" name="amount" value="" maxlength="6"
+                                   aria-required="true"/>
+                        </li>
+
+                        <li class="field text" id="make_invoice-modal-field-amount-type">
+                            <label>
+                                <input style="width:auto;margin-right: 10px" type="radio" name="amount_type"
```

Comment:
Please use CSS classes for style information (instead of inline style attributes)